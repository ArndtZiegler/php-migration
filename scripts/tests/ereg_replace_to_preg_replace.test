# Patterns with special chars
$encoded = ereg_replace("[\r\n]", '', $str);
echo $row->ID."\t".$row->Autor_ID."\t".$row->Zeit."\t".$row->Klick-Counter."\t".$row->Name."\t".ereg_replace("\n","[n]",$row->Beschreibung)."\n";
$sql=ereg_replace("[\/\n\r\\]"," ",$sql);
$text=eregi_replace("[\s[:space:]\n]*</td>[\s[:space:]\n]*<td>[\s[:space:]\n]*","	",$text);

# With variable pattern, with whitespace
$out = ereg_replace($_POST['pattern'],$_POST['replace'],$_POST['string']);
$filter[$i]=eregi_replace( $umlaute[$j][0], $umlaute[$j][1], $filter[$i]);

# Quote confusion
$sql=ereg_replace(" '","'",$sql);

# Flag confusion
$out = eregi_replace( '#blub#u', 'blabl', $in);

# Already preg
$test = preg_replace( "/blub/i" , "blab" , $satz );

# Special case: comma substitution
$money = ereg_replace( ',', '.', $sum);