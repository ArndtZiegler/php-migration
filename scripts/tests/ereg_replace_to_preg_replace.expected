# Patterns with special chars
$encoded = preg_replace("/[\r\n]/", '', $str);
echo $row->ID."\t".$row->Autor_ID."\t".$row->Zeit."\t".$row->Klick-Counter."\t".$row->Name."\t".preg_replace("/\n/","[n]",$row->Beschreibung)."\n";
$sql=preg_replace("/[\/\n\r\\]/"," ",$sql);
$text=preg_replace("/[\s[:space:]\n]*</td>[\s[:space:]\n]*<td>[\s[:space:]\n]*/i","	",$text);

# With variable pattern, with whitespace
$out = preg_replace("/" . $_POST['pattern'] . "/",$_POST['replace'],$_POST['string']);
$filter[$i]=preg_replace( "/" . $umlaute[$j][0] . "/i", $umlaute[$j][1], $filter[$i]);

# Quote confusion
$sql=preg_replace("/ '/","'",$sql);

# Flag confusion
$out = preg_replace( '#blub#ui', 'blabl', $in);

# Already preg
$test = preg_replace( "/blub/i" , "blab" , $satz );

# Special case: comma substitution
$money = preg_replace( '/,/', '.', $sum);